pkg/cloudprovider/providers/vsphere/credentialmanager_test.go

```diff
@@ -184,7 +184,7 @@ func TestSecretCredentialManager_GetCredential(t *testing.T) {
 		},
 	}
 
-	informerFactory := informers.NewSharedInformerFactory(client, controller.NoResyncPeriodFunc())
+	informerFactory := informers.NewSharedInformerFactory(client, node.NoResyncPeriodFunc())
```

Comment:
`NoResyncPeriodFunc` is a generic function for controllers, I don't think it belongs in `k8s.io/cloud-provider/node`. I think it's worth just changing this to `0` though and remove the import to `pkg/controller`.

Reply:
yes, i have think about it.
But i wonder is that we move `pkg/controller/controller_utils.go`  out of k/k one day, we may lost this change. and someone can not change `0` to `*.NoResyncPeriodFunc()' in that time, though it in test file.  WDYT?

Reply:
I personally think it's worth it. It's not ideal but we're already copying the `NoResyncPeriodFunc` method so we may miss changes from `pkg/controller` anyways. Maybe leave a comment to change it back once there's an alternative to `pkg/controller`?

Reply:
done

Reply:
forgot to remove the import to `k8s.io/cloud-provider/node` :)